.github/workflows/alert-docs.yml

```diff
@@ -33,6 +33,7 @@ jobs:
         git checkout -b update-alerts
         # Update the index to be sure git is aware of changes
         git update-index -q --refresh
+        git add site/content/docs/alerts/*
```

Comment:
Could just change L40 to add `site/*`?

Reply:
I prefer being more specific tbh - this script should only update pages in this subdirectory...

Reply:
That would be too late, that one should no longer be needed.

Reply:
Okay, include this at L40 then for consistency? Only worth adding if there's actually a change.

Reply:
I'd remove the other one but +1 to keep them the same.

Reply:
It shouldn't be too late, the line right before this updates the index so the if should be fine.

Reply:
Saying it's late because that didn't work with the SSTI rules which were just new files (and why the change).